docs/guides/theming/ThemeStructure.tsx

```diff
@@ -115,6 +116,34 @@ export const ThemeStructure = () => {
 
   const { rootId } = useTheme();
 
+  const renderClasses = (
+    value: object | string | number,
+    label: string,
+    level: number
+  ): ReactElement => {
+    return (
+      <HvAccordion
+        key={`${label}-${level}`}
+        classes={{
+          root: clsx(
+            css({ paddingLeft: levelSpacing(level) }),
+            styles.accordionRoot
+          ),
+          container: styles.accordionContainer,
+          label: styles.accordionLabel,
+        }}
+        headingLevel={1}
+        label={label}
+      >
+        <HvCodeEditor
+          height={420}
+          language="json"
+          value={JSON.stringify(value, null, "\t")}
+        />
```

Comment:
I believe it would be better to make the editor read-only with `options={{ readOnly: true }}`.